civis/civis.py

```diff
@@ -309,3 +311,21 @@ def __init__(self, api_key=None, return_type='snake',
                                    resources=resources)
         for class_name, cls in classes.items():
             setattr(self, class_name, cls(session, return_type))
+
+    @property
+    def feature_flags(self):
+        if self._feature_flags:
+            return self._feature_flags
+        me = self.users.list_me()
+        self._feature_flags = tuple(flag for flag, value
+                                    in me['feature_flags'].items() if value)
+        return self._feature_flags
+
+    def run_job(self, job_id):
```

Comment:
I was kind of intending to use this as a convenience function like the `io` functions, but I wasn't really sure where to put it since it doesn't really fall under the umbrella of `io`.